packages/communication-react/CHANGELOG.md

```diff
@@ -1,12 +1,32 @@
 # Change Log - @azure/communication-react
 
-This log was last generated on Mon, 13 Sep 2021 21:02:16 GMT and should not be manually modified.
+This log was last generated on Tue, 28 Sep 2021 19:19:18 GMT and should not be manually modified.
 
 <!-- Start content -->
 
+## [1.0.0-beta.6](https://github.com/azure/communication-ui-library/tree/@internal/react-components_v1.0.0-beta.6)
+
+Tue, 28 Sep 2021 19:19:18 GMT
+[Compare changes](https://github.com/azure/communication-ui-library/compare/@internal/react-components_v1.0.0-beta.5..@internal/react-components_v1.0.0-beta.6)
```

Comment:
Something for James to fix in the future, this should be something like: https://github.com/azure/communication-ui-library/compare/1.0.0-beta.5..1.0.0-beta.6

Reply:
Actually, not sure if this is James or Prathmesh that needs a fix.

I didn't have any entries in the changelog here at all. I started by copy-pasta'ing in the changes from all the packlets. Now you know which one I copied first.

More importantly though, I think your new link won't work either: 
![image](https://user-images.githubusercontent.com/82062616/135174676-2ad788f2-906f-415d-865d-6e67036dec01.png)

I think the problem is that we delete the release branch when we're done (so there's no https://github.com/Azure/communication-ui-library/tree/release/1.0.0-beta.5 anymore).

Reply:
It smore we haven't created the beta.6 tag, its only created at release time, e.g. comparing beta-4 and beta-5 is: https://github.com/azure/communication-ui-library/compare/1.0.0-beta.4..1.0.0-beta.5